frontend/main/components/NavigationMenu/NavigationMenu.css

```diff
@@ -71,25 +66,44 @@
   color: #ffffff;
 }
 
-@media (max-width:768px) {
-  .navigation-menu{
-    padding: 10px 20px; /* Scale down for smaller screens */
+/* Media Queries for Responsiveness */
+
+/* For Tablet Screens (768px to 1024px) */
+@media (max-width: 1024px) {
+  .navigation-menu {
+    flex-direction: column;
+  }
+  .logo-button {
+    margin-right: 0
   }
-  .logo-button{
-    margin-right: 10px; /* Reduce space for smaller screens */
+  .logo-image {
+    height: 45px; /* Scale down logo */
   }
-  .logo-image{
-    height: 40px; /* Scale down for smaller screens */
+  .nav-buttons-container {
+    gap: 20px; /* Slightly reduce the gap */
+    margin: 0;
   }
-  .nav-buttons-container{
-    flex-direction: column; /* Stack buttons for smaller screen */
+  .nav-button {
+    font-size: 14px; /* Slightly smaller font for tablet screens */
+  }
+}
+
+/* For Mobile Screens (768px and below) */
+@media (max-width: 768px) {
+  
+  .nav-buttons-container {
+    flex-direction: row; /* Keep the buttons in a row even on small screens */
     margin-right: 0;
-    gap: 15px;
+    gap: 15px; /* Reduce gap between buttons */
+    padding-bottom: 1rem;
   }
-  .nav-button{
-    flex-direction: column; /* Stack icon and text for smaller screen */
+  .nav-button {
+    flex-direction: row; /* Keep icon and text in a row */
   }
-  .nav-text{
-    font-size: 12px; /* Readable size for smaller screens */
+
+  /* Hide nav text on small screens */
```

Comment:
Cool idea! Sacrificed constraints but still our icons are predictable and follow commonly used graphic symbols.